packages/variable-editor/src/components/variables/detail/DetailContent.tsx

```diff
@@ -57,8 +56,6 @@ export const VariablesDetailContent = () => {
   const key = nodesToVariable.map(node => (node ? node.name : ''));
   const overwrites = useOverwrites(key);
 
-  const validations = useValidations(selectedVariable);
-
```

Comment:
I would not change this here. 
If you only add it to the table data you can still do it here the old way. This change is only relevant the the data table so...